pyvista/core/pointset.py

```diff
@@ -51,6 +52,41 @@
     from ._typing_core import NumpyArray
     from ._typing_core import VectorLike
 
+if TYPE_CHECKING:
```

Comment:
We can ignore missing test coverage for type-checks
```suggestion
if TYPE_CHECKING:  # pragma: no cover
```